api_app/playbooks_manager/controller.py

```diff
@@ -0,0 +1,129 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+
+from celery import chord, group
+from django.conf import settings
+
+from api_app.analyzers_manager.dataclasses import AnalyzerConfig
+from api_app.connectors_manager.dataclasses import ConnectorConfig
+
+from ..models import Job
+from .dataclasses import PlaybookConfig
+
+logger = logging.getLogger(__name__)
+
+
+def start_playbooks(
```

Comment:
coverage report states that this function is not tested at all. Could you add a test for this?

Reply:
let me write a general one for this.